gitreleasemanager.yaml

```diff
@@ -5,6 +5,7 @@ issue-labels-include:
 - Bug
 - Enhancement
 - Documentation
+- Deprecate
```

Comment:
@vexx32 the order of the entries in this part of the configuration file also controls the order of the entries on the release notes.  I think we would want this higher up in the order, no?  Perhaps beside `Breaking Change`?  Thoughts?

Reply:
I have taken the liberty of moving this slightly further up in the order of things.